class PreconditionFail(Exception):
    """This exception is thrown when a test failed one of its precondition and cannot test what it was meant for."""
